netdata-installer.sh

```diff
@@ -934,6 +934,34 @@ if [ "$have_autotools" ]; then
   fi
 fi
 
+# function to extract values from the config file
+config_option() {
+  section="${1}"
+  key="${2}"
+  value="${3}"
+
+  if [ -s "${NETDATA_PREFIX}/etc/netdata/netdata.conf" ]; then
```

Comment:
```suggestion
  if [ -x "${NETDATA_PREFIX}/usr/sbin/netdata" ] && [ -r "${NETDATA_PREFIX}/etc/netdata/netdata.conf" ]; then
```

- We need to know that the command we0re trying to run actually exists and is executable before invoking it. It’s entirely possible for the config file to exist but the netdata binary to not exist (for example, reinstalling after previously uninstalling without removing config files).
- We need to check for readability of the config file, otherwise we will potentially spill error messages that are meaningless to users. Notably, checking for file size does not guarantee that the file is readable, just that the parent directory is readable and traversable.
- We can probably safely skip checking size, as the agent will just spit out a default value then instead of giving us trouble.